athf/core/attack_matrix.py

```
@@ -128,4 +490,67 @@ def get_sorted_tactics() -> List[str]:
+
+
+def is_using_stix() -> bool:
+    """Check if using live STIX data."""
+    return _get_provider().is_stix()
+
+
+# ---------------------------------------------------------------------------
+# Module-level __getattr__ (PEP 562) for backward compatibility
+# ---------------------------------------------------------------------------
+
+def __getattr__(name: str) -> Any:
```

Ruff (0.15.6)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `__getattr__`

(ANN401)